<launch>
  <!-- node pkg="ftag2" type="ros_cam_testbench" name="ros_cam_testbench" output="screen" launch-prefix="gdb -ex run - -args" -->
  <node pkg="ftag2" type="ros_cam_testbench" name="ros_cam_testbench" output="screen">
    <param name="source" type="str" value="/home/thalassa/anqixu/Desktop/test2.png" />
    <param name="source" type="str" value="/home/thalassa/anqixu/Desktop/source.png" />
    <param name="source" type="str" value="/localdata/anqixu/AQUA_TAG_BAGS/contrast.jpg" />
    <param name="camMatrix" type="str" value="/home/dacocp/Dropbox/catkin_ws/Intrinsics.xml" />
    <param name="distCoef" type="str" value="/home/dacocp/Dropbox/catkin_ws/Distortion.xml" />
    <param name="source" type="str" value="" />

    <param name="waitkey_delay" value="30" />
    <param name="save_img_dir" value="/localdata" />
    
    <!-- The following are recommended parameter values line detector using connected components -->
    <param name="sobelThreshHigh" value="100" />
    <param name="sobelThreshLow" value="30" />
    <param name="sobelBlurWidth" value="3" />
    <param name="houghEdgelThetaMargin" value="20.0" />
    <param name="houghMinAccumValue" value="50.0" /> <!-- minimum number of edgels within each connected component -->
    <param name="houghMinSegmentLength" value="15.0" />
    
    <param name="houghMaxSegmentGap" value="7.0" />
    <param name="numberOfParticles" value="50" />
    <param name="position_std" value="0.5" />
    <param name="orientation_std" value="0.5" />
    <param name="position_noise_std" value="0.3" />
    <param name="orientation_noise_std" value="0.2" />

  </node>
</launch>
